dotcom-rendering/src/components/Section.tsx

```diff
@@ -305,6 +305,9 @@ export const Section = ({
 								borderColour={
 									borderColour ?? overrides?.border?.container
 								}
+								textColour={
+									overrides?.text?.container ?? neutral[7]
```

Comment:
Is there a reason we're setting the default colour here and not in the component itself? I feel like in the past we've generally preferred setting a default colour in the css itself such as https://github.com/guardian/dotcom-rendering/pull/7891/files#diff-ddbc9e36ed42e5291d5d936698dae0577718b955269fe57e8363ac3adf1b8591R31